File: MPI/mpicheck/tests/functions/1.0/MPI_Testsome.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2
       INTEGER var_3(10)
       INTEGER var_4(10,10)
       INTEGER var_5
        call mpi_testsome(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_testsome(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Testsome.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Request var_1[2];
    int *var_2;
    int var_3[2];
    MPI_Status var_4[2];
    int ret;
    /* calls */
    ret = MPI_Testsome(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Testsome(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
